news/2017-08-17-happystick-summer-tour-concludes.md

```diff
@@ -0,0 +1,50 @@
+
+[Full match information](https://osu.ppy.sh/community/matches/35249726)
+
+<p align="center"><iframe src="https://player.twitch.tv/?video=v167003412&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe></p>
+
+From the [62500 - 80000 bracket](http://challonge.com/bqmca1h3) came one of the most exciting matches of the day, if not the entire tournament. On the surface it looked similar to the 12000 - 18000 matchup: the Winners Bracket was won in impressive style by [snowdeep](https://osu.ppy.sh/u/8614428), while [Dreamtwolf](https://osu.ppy.sh/u/6138845) dropped to the Losers Bracket via tiebreaker in the first round then fought all the way through, winning another tiebreaker in the Losers Final to secure his ticket to the Grand Final. And what a Grand Final it was. In the first best of 11, the players went back and forth until 4-4, their abilities tested on Nomod, Hidden, and Double Time picks. The last three maps of this series would prove to be the highlight of the match. On [Krewella - Surrender the Throne](https://osu.ppy.sh/b/1037972), both players held combo for over half the map before Dreamtwolf missed just before a break in the map; seeing his chance, snowdeep took the opportunity and built up a lead that proved to be insurmountable. Next up was [Reol - Asymmetry](https://osu.ppy.sh/b/658387), where Dreamtwolf put on a fantastic performance, despite HappyStick invoking his famous caster curse [four times in a row](https://www.twitch.tv/videos/167003412?t=56m40s), to send the set to a tiebreaker. [Treyarch Sound - Abracadavre](https://osu.ppy.sh/b/882897) would be the scene, both players electing to play with Hidden. Over the course of the map’s near-six-minute duration the players would go back and forth constantly as the drama rose to a crescendo with Dreamtwolf coming out on top by a mere 280,000 points, taking the match to a second best of 11. The reset match would turn out to be all Dreamtwolf all the time as he put on an utterly dominant performance including three Full Combos showing off his incredible Hidden consistency on [senya - Kanjou Chemistry (Drum 'n' Bass Remix)](https://osu.ppy.sh/b/258252) and [Madotsuki@ - Ikanaide](https://osu.ppy.sh/b/703905) to race to a 6-2 win and the championship victory. 
```

Comment:
new paragraph at 'And what a Grand Final it was'

new paragraph at 'In the first best of 11'
(this may seem a little odd but you've got good cadence there with the emphasis on grand final, and i think it's perfectly acceptable to note that)

new paragraph at 'Next up was Reol'

new paragraph at 'The reset match would turn out'